command/login.go

```diff
@@ -240,6 +240,10 @@ func (c *LoginCommand) Run(args []string) int {
 		c.UI.Warn(wrapAtLength("A login request was issued that is subject to "+
 			"MFA validation. Please make sure to validate the login by sending another "+
 			"request to sys/mfa/validate endpoint.") + "\n")
+
+		// We return early to prevent success message from being printed
+		c.checkForAndWarnAboutLoginToken()
```

Comment:
I am not sure if this call is needed here. Cause at this point, there is no Token to be returned as the user would need to validate the MFA. What do you think? Worth discussing with the devex team on this.

Reply:
The conversation with devex: https://hashicorp.slack.com/archives/C025608TAR0/p1652448116320469 seemed to support this behaviour - in essence, even if there is no token set, the user's primary goal with this command is to set a token, so warning them still makes sense